File: src/main/java/ru/igrey/dev/entity/DealHistoryEntity.java
package ru.igrey.dev.entity;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@AllArgsConstructor
public class DealHistoryEntity {
    private Long id;
    private Long dealId;
    private Long userId;
    private Long oldStatusId;
    private Long newStatusId;
    private String comment;
    private String event;
    private Long sourceTokenId;

    public String toString() {
        return "STI:" + sourceTokenId + "; userId: " + userId + ";\n" + oldStatusId + " -->" + event + "--> " + newStatusId;
    }
}

File: src/main/kotlin/ru/igrey/dev/entity/DealHistoryEntity.kt
package ru.igrey.dev.entity

data class DealHistoryEntity(
    val id: Long,
    val dealId: Long,
    val userId: Long,
    val oldStatusId: Long,
    val newStatusId: Long,
    val comment: String,
    val event: String,
    val sourceTokenId: Long
) {
    override fun toString() = "STI:$sourceTokenId; userId: $userId;\n$oldStatusId -->$event--> $newStatusId"
}